src/browser/js/DesktopBridge.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2015-2016 Yuya Ochiai
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import EventEmitter from 'events';
+
+import DesktopConnector from './DesktopConnector';
+
+// placeholder bridge object
+window.desktopBridge = {
+  ready: false,
+};
+
+/**
+ * Provides the bridging infrastructure to connect the desktop and webapp together
+ */
+export default class DesktopBridge extends EventEmitter {
+  constructor() {
+    super();
+
+    this.ready = false;
+
+    this.desktop = null;
+    this.webapp = null;
+
+    window.desktopBridge = this;
```

Comment:
Why use a global instance of the `DesktopBridge`? Couldn't this be something that's owned by the `MattermostView`?

Reply:
I believe we resolved this question when we talked last week?